rpcs3/rpcs3qt/rpcn_settings_dialog.cpp

```diff
@@ -800,6 +800,7 @@ void rpcn_account_edit_dialog::change_password()
 
 			QMessageBox::information(this, tr("Password Reset Token Sent!"), tr("The reset password token has successfully been sent!"), QMessageBox::Ok);
 		}
+		[[fallthrough]];
```

Comment:
why does this fallthrough?

Reply:
if no => send token and then ask for token
if yes => ask for token
I could technically move the yes code out of the switch but it's fine as it is.